server/routes/induction/update/workInterestTypesFormValidator.ts

```diff
@@ -0,0 +1,54 @@
+import type { WorkInterestTypesForm } from 'inductionForms'
+import type { PrisonerSummary } from 'viewModels'
+import formatErrors from '../../errorFormatter'
+import WorkInterestTypeValue from '../../../enums/workInterestTypeValue'
+
+export default function validateWorkInterestTypesForm(
+  workInterestTypesForm: WorkInterestTypesForm,
+  prisonerSummary: PrisonerSummary,
+): Array<Record<string, string>> {
+  const errors: Array<Record<string, string>> = []
+
+  errors.push(...formatErrors('workInterestTypes', validateWorkInterestTypes(workInterestTypesForm, prisonerSummary)))
+  errors.push(
+    ...formatErrors('workInterestTypesOther', validateWorkInterestTypesOther(workInterestTypesForm, prisonerSummary)),
+  )
+  return errors
+}
+
+const validateWorkInterestTypes = (
+  workInterestTypesForm: WorkInterestTypesForm,
+  prisonerSummary: PrisonerSummary,
+): Array<string> => {
+  const errors: Array<string> = []
+
+  const { workInterestTypes } = workInterestTypesForm
+  if (!workInterestTypes || workInterestTypes.length === 0 || containsInvalidOptions(workInterestTypes)) {
```

Comment:
The logic is simpler here, compared to other validators, since `NONE` is not an option.